File: BACKEND/PickleballCourtBookingSystem/PickleballCourtBookingSystem.Core/Entities/Cancellation.cs
namespace PickleballCourtBookingSystem.Core.Entities

{
    public class Cancellation
    {
        public Guid? Id { get; set; }
        public DateTime? TimeCancel { get; set; }
        public string? Reason { get; set; }
        public Guid? BookingId { get; set; }
    }
}

File: AndroidApp/app/src/main/java/com/maxholmes/androidapp/data/model/Cancellation.kt
package com.maxholmes.androidapp.data.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize
import java.time.LocalDateTime

@Parcelize
data class Cancellation(
    var id: String,
    var timeCancel: LocalDateTime,
    var bookingId: String,
    var reason: String
) : Parcelable